docs/schema.md

```diff
@@ -475,6 +475,13 @@ vulnerabilities as `aliases` would mean that they are all identical (due to the
 symmetry/transitivity of `aliases`), not that one release fixes multiple
 (distinct) vulnerabilities.
 
+Aliases should **not** be used to refer to vulnerabilities in packages upstream
```

Comment:
The previous point about bundling now feels a bit redundant with this definition framed around upstream/downstream. 

could we somehow merge the two? i.e. start with the definition you've provided, and then also use the bundling many different vulnerabilities bit as an additional example. Something like:

Aliases should **not** be used to refer to vulnerabilities in packages upstream 
or downstream in a software supply chain from the given OSV record's affected
package(s). For example, if a CVE describes a vulnerability in a language
library, and a Linux distribution package contains that library and therefore
publishes an advisory, the distribution's OSV record must not list the CVE ID as
an alias. Similarly, distributions often bundle multiple upstream vulnerabilities into a single record. `related` should be used in these cases. 

CC @michaelkedar for thoughts too.

Reply:
I like that — adjusted in 21cbf35